cmd/create/cluster/cmd.go

```diff
@@ -712,11 +712,12 @@ func run(cmd *cobra.Command, _ []string) {
 			r.Reporter.Errorf("Expected a valid OpenShift version: %s", err)
 			os.Exit(1)
 		}
-	}
-	version, err = validateVersion(version, versionList, channelGroup, isSTS, isHostedCP)
-	if err != nil {
-		r.Reporter.Errorf("Expected a valid OpenShift version: %s", err)
-		os.Exit(1)
+	} else {
```

Comment:
Any specific reason for the else?

Reply:
there is some duplication in the validation, but after double checking we can't just skip the second round. I removed it from the PR, it can be improved separately